### ISSUE TITLE
tag times in ABFs with discontinuous sweeps

### ISSUE THREAD

**Issue by swharden** [swharden/pyABF#50]
I found a bug in [ABF-Tag-Editor](https://github.com/swharden/ABF-Tag-Editor) which makes me think tag times in pyABF might be wrong in ABFs with space between their sweeps (non-minimum inter-sweep interval?)

Find an ABF with tags which meets this criteria and confirm.

**Comment by swharden** [comment]
I added a demo ABF to test against. It should be 5 minutes long with a comment tag at 3 minutes. https://github.com/swharden/pyABF/commit/301d0ace251af324acfeac5bace754aa75dbe35d

```
Trial length: 1 run; 60 sweeps of 100.0 ms (2000 samples).
Minimum time between run starts.
5.000 s between sweep starts.
Sampling rate:  20.00 kHz (interval 50.00 µs).
```

**Comment by swharden** [comment]
Fixed by 860af4a2ebc219c2108359abb26ca86e790c55bf

Output of https://github.com/swharden/pyABF/blob/master/dev/python/2018-12-08%20discontinuous%20sweeps.py:

```
ABFs where sweep length != sweep interval:
interval        length          ABF
66.00           6.19            c:\Users\scott\Documents\GitHub\pyABF\data\abfs\171117_HFMixFRET.abf
20.00           1.50            c:\Users\scott\Documents\GitHub\pyABF\data\abfs\18711001.abf
20.00           1.50            c:\Users\scott\Documents\GitHub\pyABF\data\abfs\18713001.abf
5.00            0.10            c:\Users\scott\Documents\GitHub\pyABF\data\abfs\2018_11_16_sh_0006.abf
5.00            1.00            c:\Users\scott\Documents\GitHub\pyABF\data\abfs\File_axon_5.abf
20.00           1.00            c:\Users\scott\Documents\GitHub\pyABF\data\abfs\File_axon_6.abf
10.00           4.01            c:\Users\scott\Documents\GitHub\pyABF\data\abfs\File_axon_7.abf
```
